addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -28,7 +28,7 @@
                     <button class="btn btn-link text-action" t-att-class="{ 'o-active': state.isSearchOpen }" aria-label="Search Messages" title="Search Messages" t-on-click="onClickSearch">
                         <i class="oi oi-search" role="img"/>
                     </button>
-                    <button t-if="props.hasAttachmentPreview and state.thread.attachmentsInWebClientView.length" class="btn btn-link text-action" t-on-click="popoutAttachment">
+                    <button t-if="props.hasAttachmentPreview and state.thread.attachmentsInWebClientView.length" class="btn btn-link text-action o_chatter_attachment_control" t-on-click="popoutAttachment">
                         <i class="fa fa-window-restore" aria-hidden="Pop out Attachments" title="Pop out Attachments"/>
                     </button>
```

Comment:
No need to add a classname and pollute template for only tests: you can `contains("button:contains('Pop out Attachments')")` in test